index.html

```diff
@@ -10630,8 +10630,19 @@ <h2 id="AudioWorklet-Examples">
             <dd>
               <p>
                 This method is DEPRECATED. It is equivalent to setting
-                <code>orientationX</code>, <code>orientationY</code>, and
-                <code>orientationZ</code> AudioParams directly.
+                <code>orientationX.value</code>,
+                <code>orientationY.value</code>, and
+                <code>orientationZ.value</code> attribute directly, with the
+                <code>x</code>, <code>y</code> and <code>z</code> parameters,
+                respectively.
+              </p>
+              <p>
+                Consequently, if any of the <code>orientationX</code>,
+                <code>orientationY</code>, and <code>orientationZ</code>
+                <a>AudioParam</a> have an automation curve set using <a href=
+                "#dom-audioparam-setvaluecurveattime">setValueCurveAtTime()</a>
```

Comment:
I think the link should be `<a data-link-for="AudioParam">setValueCurveAtTime()</a>`.  Maybe parens outside the tag?